src/modals/select-baker-modal/select-baker-modal.tsx

```diff
@@ -77,24 +114,42 @@ export const SelectBakerModal: FC = () => {
         </View>
         <View style={styles.upperContainer}>
           <Text style={styles.infoText}>The higher the better</Text>
+          <View style={styles.sortSelector}>
+            <Text style={styles.sortByLabel}>Sort by</Text>
+            <Dropdown<BakersSortFieldEnum>
+              title="Sort bakers by:"
+              list={bakersSortFieldsOptions}
+              equalityFn={(a, b) => a === b}
+              renderValue={({ value }) => (
+                <View style={styles.selectedBakerWrapper}>
+                  <Text style={styles.selectedBakerSortField}>
+                    {bakersSortFieldsLabels[value ?? BakersSortFieldEnum.Rank]}
+                  </Text>
+                  <Icon size={formatSize(24)} name={IconNameEnum.TriangleDown} />
+                </View>
+              )}
+              renderListItem={({ item }) => (
+                <Text style={styles.bakersSortListItem}>
+                  {bakersSortFieldsLabels[item ?? BakersSortFieldEnum.Rank]}
+                </Text>
+              )}
+              onValueChange={value => setSortValue(value ?? BakersSortFieldEnum.Rank)}
+              style="nativeLike"
+              contentHeight={formatSize(216)}
+              value={sortValue}
+            />
```

Comment:
you don't need to create `"nativeLike"` dropdown.  
This's already implemented as a separate component.  
Use `BottomSheet` and `BottomSheetActionButton` instead. (ex. in `ManageHdAccounts`)